docs/source/getting_started/quick_start.md

```diff
@@ -1,8 +1,8 @@
 # 🚀 Quick Start
 
-Start training your model with the Streaming dataset in a couple of steps!
+Start training your model with the Streaming dataset in a few steps!
 
-- Convert your raw dataset into a supported streaming format, for example, `mds` format.
+1. Convert your raw dataset into one of our supported streaming format, for example, `mds` (Mosaic Data Shard) format.
```

Comment:
formats

Reply:
Done in https://github.com/mosaicml/streaming/pull/33/commits/a634dbdbd8039c0755687d3cc8172cba15c977ed.